rdr_service/services/ghost_check_service.py

```diff
@@ -3,11 +3,15 @@
 
 from sqlalchemy.orm import Session
 
+from rdr_service.config import GAE_PROJECT
```

Comment:
The GhostCheckService class is used by a script as well as the cron job. Can you have them pass in the project id, rather than have the class try to determine which one it is on its own? That way the service class can still be used by the script and have the rebuild tasks created on the target environment.

Reply:
Updated.   FYI:   We're going to be deprecating all of this rebuild task dispatching when the new RDR-PDR pipeline is implemented

Reply:
I could be mistaken, but I think it's still needed. If `dispatch_participant_rebuild_tasks` is called without the `project_id` parameter it will use this same concept and try to determine the target environment itself. So in the same way it would find the project as localhost when the ghost check script is being run against something like prod. What I've seen before is that letting it find localhost will have it try to rebuild everything into the local database. And as far as I could tell at the time, that ends up not making it to the server for PDR.

The `build_locally` parameter would need to be set to False to avoid having that happen too.

Reply:
Updated

Reply:
Final (hopefully) update:   revised the logic of how the `build_locally` flag is handled in the dispatch tasks method so that it's not required unless you really want to override the cloud task and force a local build.